src/Game.js

```diff
@@ -0,0 +1,67 @@
+class Game {
+  constructor() {
+    this.score = 0;
+    this.frameCounter = 0;
+    this.MAX_FRAME = 10;
+  }
+
+  bowlFrame(roll1, roll2) {
+    if (this.isGameOver()) {
+      throw new Error("game over")
+    }
+    this.frame = [roll1, roll2]
+    this.addScore(roll1, roll2)
+
+    this.isFrameASpareOrStrike(roll1, roll2)
+
+    ++this.frameCounter
+  }
+
+  addScore(roll1, roll2) {
+    this.checkLastFrameStatus()
+    this.score += (roll1+roll2)
+  }
+
+  showScore() {
+    return this.score;
+  }
+
+  isGameOver() {
+    if (this.frameCounter === this.MAX_FRAME) {
```

Comment:
```suggestion
    return  this.frameCounter === this.MAX_FRAME 
```

should do the trick

Reply:
Oh nice, good refactor there. Thanks for the tip!